game/action.go

```diff
@@ -7,13 +7,15 @@ import (
 type Action struct {
 	Type          ActionType
 	Card          *Card
+	CostTarget    *Permanent
```

Comment:
what is a "cost target"? without looking through your code I am unable to figure it out

Reply:
i mean, costs... don't have targets?